### PR TITLE
Remove cpp examples from readme

### PR DESCRIPTION
**Pull request by Corallo**
### Describe your changes

Removing deprecated cpp examples from README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,12 +93,13 @@ The ACAP Runtime service provides the following APIs:
...
-  [Python][minimal-ml-inference] and [C++][object-detector-cpp].
+  [Python][minimal-ml-inference].
 - Parameter API - Provides gRPC read access to the parameters of an Axis device.
   There are usage examples available for the Parameter API written in
-  [Python][parameter-api-python] and [C++][paramter-api-cpp].
-- Video capture API - Enables capture of images from a camera. No usage examples
-  for this API exist yet.
+  [Python][parameter-api-python].
+- Video capture API - Enables capture of images from a camera.
+  There are usage examples available for the Parameter API written in
```
Comment (madelen-at-work) [inline README.md:101]: I think you can remove this line - we don't list every example. Alternatively add it under the Examples section instead

**Review by madelen-at-work — COMMENTED** [review]
Looks good but please check the line on the Parameter API example

File: README.md (hunk 1 above)
Comment (Corallo) [reply README.md:101]: Do you mean the line:
"There are usage examples available for the Parameter API written in [Python][opencv-qr-decoder-python]." ?

Comment (madelen-at-work) [reply README.md:101]: yes

**Pull request merged**